### ISSUE TITLE
missing link to updates.html5rocks.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#731]
```
Please describe the issue:
there's no link from top page to http://updates.html5rocks.com/

Is this intentional or just forgotten?
should this be fixed? or can wait for ver.2?
```

Original issue reported on code.google.com by `agek...@google.com` on 30 Nov 2011 at 5:36

**Comment by GoogleCodeExporter** [comment]
```
Will not fix. Waiting on v2.
```

Original comment by `ericbide...@html5rocks.com` on 2 Dec 2011 at 4:15
- Changed state: **WontFix**
